### ISSUE TITLE
[Feature Request] Restructuring of the publication page for a better overview

### ISSUE THREAD

**Issue by CMR248** [nfdi4plants/archigator-backend#5]
- [x] Headline: "Welcome to the ARC Data Publication Service"
- [x] Visually separate the headline from the rest of the web page with a thick separator line
- [x] Dividing the remaining webpage into the publication-describing part (left, wide, white) and the administrative part (right, less wide, grey)
- [x] Concerning the publication-describing part, the layout could look like common publication standards (e.g. Zenodo)
- DATA PUBLICATION
- Date
- Name of the Publication Server (ARChive)
- Investigation Title (which is not necessarily the title of the Repository)
- List of Authors including
** Mailadresses
** Affiliation
** ORCiD (with Symbol)
- Investigation Description 
- [x] The administrative part includes function buttons for publishing and hint fields about the underlying GitLab repository

- No User as this might be confusing
- Publish ARC Button at the top
- In order that all authors are listed, it would be good to query exactly that again via a checkbox.
- Instead of displaying Project, display "Source of this Data Publication is <project name>".
- The unsuccessful critical tests should be displayed underneath, with a text like "Project not ready to publish" and afterwards "Make sure that the following criteria are met:". 
** The successful ones should be omitted, as they tend to cause confusion.
** An explanation of what should be adjusted to correct the pipeline would be nice
** Here one could refer to an area in the knowledge base, which goes into detail concerning individual messages, or @Brilator 
- [x] If Terms and Conditions/Author List checkboxes are not checked, the following should happen
** Link Terms and Conditions properly
- [x] A "back" button would be desirable to return to the repo

For a better visualization and estimation of the font sizes find images below

**Restructured Page**
![image](https://github.com/nfdi4plants/archigator/assets/74731649/b01b88ed-784e-4e38-bdb2-5cb903c191a7)

**Terms and Conditions/Authors list weren´t checked**
![image](https://github.com/nfdi4plants/archigator/assets/74731649/7d391352-af3d-4e98-b69e-6273939e2bd9)

**Tests failed**
![image](https://github.com/nfdi4plants/archigator/assets/74731649/f0eed6a5-0e13-4983-b440-4d3433ef4c8e)

**(Part of) Zenodo Publication Page**
![image](https://github.com/nfdi4plants/archigator/assets/74731649/936bbf39-149c-471c-b030-7a8d900821ab)

**Comment by Zerskk** [comment]
@omaus @HLWeil orcid and affiliation must be added to metadata.json , so that this items can be displayed on the webpage

**Comment by omaus** [comment]
> @omaus @HLWeil orcid and affiliation must be added to metadata.json , so that this items can be displayed on the webpage

@Zerskk 

We will provide email address, ORCID, and affiliation ***name*** but cannot provide affiliation ***ID*** since this has to be done by Invenio:
![image](https://github.com/nfdi4plants/archigator/assets/47781170/635edcc2-1429-44f1-8c66-9a9ec472748f)

To me, it seems like you'd need to implement a parser that crawls a CV for a specific name and then fills in the ID if the name exists, else... I don't know, creates a random ID and updates the CV?

CC: @kMutagene

**Comment by kMutagene** [comment]
For context, here is an overview of the current (as of 2023-07-11) validation and metadata creation pipeline:

## Repos:
- [arc-export](https://github.com/nfdi4plants/arc-export)
- [arc-validate](https://github.com/nfdi4plants/arc-validate)
- [arc-to-invenio](https://github.com/nfdi4plants/arc-to-invenio)

## Jobs:

Note that `arc-to-invenio` only runs when `arc-validate` succeeds without critical tests failing

```mermaid
flowchart LR

arc[ARC on GitLab]

arc-export[["arc-export"]]
arc-validate[["arc-validate"]]
arc-to-invenio[["arc-to-invenio"]]

arc_json("arc.json")
arc_validate_result("arc-validate-results.xml")
arc_quality("arc-quality.svg ")
invenio_metadata("metadata.json")

arc --> arc-export --> arc_json

arc --> arc-validate 
arc-validate --> arc_validate_result
arc-validate --> arc_quality

arc --> arc-to-invenio --> invenio_metadata
``` 

## Files:

- `arc.json`:  contains the full arc metadata (e.g. all isa files in the arc in json format). Intended to be used for 'inspecting the arc' without downloading it, e.g. for a preview on the invenio page when the arc is published. 
- `arc-validate-results.xml`:  junit-formatted file containing the results of all tests performed on the arc by the `arc-validate` tool.
- `arc-quality.svg`:  Indicates the quality of the arc based on passed tests, e.g. ![](https://raw.githubusercontent.com/nfdi4plants/arc-validate/main/test/arc-quality.svg)
- `metadata.json`: Invenio metadata record containing necessary information extracted from the arc by `arc-to-invenio` to create a bibliographic record on invenio. See also [arc-to-invenio docs](https://github.com/nfdi4plants/arc-to-invenio#supported-fields) for a list of supported fields.

## Difference between `arc.json` and `metadata.json`

`metadata.json` contains only the fields required for creaing a json file adhering to the invenio rdm metadata [json schema](https://inveniordm.docs.cern.ch/reference/metadata/#metadata)

`arc.json` is the serialized full arc datamodel. It is a superset of `metadata.json`, which in contrast only contains invenio-relevant metadata. Note that properties are also named differently in these two files, e.g. 

`arc.json`:
```json
"people": [
    {
      "lastName": "Weil",
      "firstName": "Lukas",
      "email": "weil@nfdi4plants.org",
      "affiliation": "RPTU",
      "comments": [{ "name": "Investigation Person ORCID", "value": "orcid1" }]
    }
]
```

vs. `metadata.json`:

```json
"creators": [
    {
      "person_or_org": {
        "type": "personal",
        "name": "Weil, Lukas",
        "given_name": "Lukas",
        "family_name": "Weil",
        "identifiers": [
          { "scheme": "email", "identifier": "weil@nfdi4plants.org" },
          { "scheme": "orcid", "identifier": "orcid1" }
        ]
      },
      "affiliations": [{ "name": "RPTU" }]
    }
]
```

So in short, every information contained in `metadata.json` is also in `arc.json`, but `arc.json` can contain more metadata that is not relevant for the invenio record json schema.

~~I'd suggest retrieving all metadata that are not directly related to the invenio metadfata record from `arc.json`.~~

`metadata.json` should contain all information needed to fill the forms.

**Comment by Zerskk** [comment]
all tasks implemented, further review required
